doc/Advanced.md

```diff
@@ -1486,6 +1486,18 @@ The behaviour of the check can be inverted by setting the customparameter "mustI
         }
 ```
 
+The number of cycles during which we tolerate increasing (or not respectively) the number of entries can be set with the custom parameter `nbBadCyclesLimit`:
+```
+        "extendedCheckParameters": {
+          "default": {
+            "default": {
+              "nbBadCyclesLimit": "3",
```

Comment:
```suggestion
              "nBadCyclesLimit": "3",
```
I would propose to use just `n` for "number", which is seems more common in our codebase.

Reply:
agreed, I will commit the change myself because there is a second place.